unittests/test_schedulers.py

```diff
@@ -530,6 +530,52 @@ def test_submit_timelimit(self):
         pytest.skip("PBS minimum time limit is 60s")
 
 
+class TestTorqueJob(_TestJob, unittest.TestCase):
+    @property
+    def sched_name(self):
+        return 'torque'
+
+    @property
+    def launcher_name(self):
+        return 'local'
+
+    @property
+    def sched_configured(self):
+        return fixtures.partition_with_scheduler('torque') is not None
+
+    def setup_user(self, msg=None):
+        super().setup_user(msg='Torque not configured')
+
+    def test_prepare(self):
+        self.setup_job()
+        self.testjob.options += ['-l mem=100GB', 'haswell']
+        super().test_prepare()
+        num_nodes = self.testjob.num_tasks // self.testjob.num_tasks_per_node
+        num_cpus_per_node = (self.testjob.num_cpus_per_task *
+                             self.testjob.num_tasks_per_node)
+        expected_directives = set([
+            '#PBS -N "testjob"',
+            '#PBS -l walltime=0:5:0',
+            '#PBS -o %s' % self.testjob.stdout,
+            '#PBS -e %s' % self.testjob.stderr,
+            '#PBS -l nodes=%s:ppn=%s:haswell' % (num_nodes, num_cpus_per_node),
+            '#PBS -l mem=100GB',
+            '#PBS -q %s' % self.testjob.sched_partition,
+            '#PBS --gres=gpu:4',
+            '#DW jobdw capacity=100GB',
+            '#DW stage_in source=/foo'
+        ])
+        with open(self.testjob.script_filename) as fp:
+            found_directives = set(re.findall(r'^\#\w+ .*', fp.read(),
+                                              re.MULTILINE))
+
+        self.assertEqual(expected_directives, found_directives)
```

Comment:
Use pytest syntax here:

```suggestion
        assert expected_directives == found_directives
```